integration_tests/requirements.txt

```diff
@@ -3,4 +3,5 @@ dbt-bigquery>=1.0.0
 dbt-redshift>=1.0.0
 dbt-postgres>=1.0.0
 dbt-spark>=1.0.0
-dbt-spark[PyHive]>=1.0.0
\ No newline at end of file
+dbt-spark[PyHive]>=1.0.0
```

Comment:
Shouldn't requirements also have a min of <1.3.0 for each DB type like the other packages?

Reply:
good catch! updated